blocks/article-hero-video/article-hero-video.css

```diff
@@ -0,0 +1,136 @@
+div.article-hero-video-container {
+    overflow: hidden;
+    padding-top: 0;
+}
+
+div.article-hero-video-overlay{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100%;
+    z-index: 1;
+}
+
+div.article-hero-video {
+    width: 100vw;
+    height: auto;
+    aspect-ratio: 16/9;
+    border: none;
+    margin: 0;
+    animation-timing-function: linear;
+    display: block;
+    transform: scaleX(1.1) scaleY(1.1)
+}
+
+div.article-hero-video-wrapper {
+    margin: 0;
+}
+
+
+div.article-hero-video-title-container {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: auto;
+    height: auto;
+    min-width: 80%;
+    padding: 0;
+    margin: 0;
+    color: #fff;
+}
+
+h1.article-hero-video-title {
```

Comment:
try to put the prefix `.article-hero-video` before each selector, then you can put simpler sub-selectors like `.title`. This ensures that there is absolutely no overlap with other blocks.